docs/custom-metric-filter.md

```diff
@@ -164,3 +171,151 @@ This can be done by deleting the Otel Collector deployment.
     
     - `kubectl get all -n config-management-monitoring` should show all objects running with no error.
     - `kubectl get cm -n config-management-monitoring` should include a new ConfigMap named `otel-collector-custom`
+
+## Sample Configmap
```

Comment:
Won't having this here mean it needs to be updated every time the real one is updated? Won't that cause it to quickly become out of date? Would it be better to just link to the real one?

Reply:
Forgot about this PR.. The ConfigMap won't go stale, once one with name `otel-collector-custom` exists the otel collector deployment will always load the custom one. I'm not sure what the `real one` is implying, could you elaborate?

Reply:
I mean, this "sample" will drift from what's in the code as the default config. So if people copy this one, it might not match what's in the code.

Reply:
Not sure if this change is necessary anymore, it was for users with version v1.14.1, when they want to modify their ConfigMap they will find the format is all off. This problem is fixed in later versions and I concur adding a copy of config here would cause drift. The link to the actual config is given on line70 as well.

If @haiyanmeng also agrees I'll close this PR.